Src/Plugins/Security/Security.SharedStrings.txt

```diff
@@ -23,15 +23,23 @@
 
 # $FacebookAppCredentials=(?si)(((facebook|fb)(.{0,20})?[`'"\s>](?P<id>[0-9]{13,17})[^0-9].{1,500}?)|((facebook|fb)(.{0,20})?[`'"\s>](?P<secret>[0-9a-f]{32})([^0-9a-f]?)).{1,500}?){2}
 
-  $SEC101/001.HttpAuthorizationRequestHeader=(?i)(?P<scheme>http|ftp|https):\/\/(?P<host>[\w_.-]{1,200})(?P<path>[\w.,@?^=%&:\/~+#-]*[\w@?^=%&\/~+#-])?(.|\n){0,100}?authorization[,\[:= "']+(basic)[\s\r\n]{0,10}(?P<secret>[^'"><;\s]{1,500})
+# Intrafile analysis
+  $SEC101/004.FacebookAppCredentialsId=(?si)facebook.{1,50}\b(?P<id>[0-9]{15})([^0-9]|$)
```

Comment:
>FacebookAppCredentialsId [](http://example.com/codeflow?start=14&length=24)

Please review these reauthored regexes carefully to form an opinion on them. We should be *very* consistent in the way we author them. Note that one specific of the facebook rewrite is that we require an explicit prefix of some kind with the term 'facebook' in it before we attempt to retrieve what is presumably a secret. We do this because the pattern for the facebook secret is so generic.